src/components/ImagesCarousel.js

```diff
@@ -130,8 +130,10 @@ export const ImagesCarousel = ({
 
   if (!data || data.length === 0) return null;
 
+  const sliderSettings = data[0]?.sliderSettings || {};
+
   // filter data for present items and items with active date/time periods
-  const carouselData = data.filter((item) => item && isActive(item));
+  const carouselData = data.filter((item) => item && isActive(item) && !item.sliderSettings);
```

Comment:
the comment is now missing information about the newly added condition

Reply:
I updated the commit.

Reply:
i don't get it. i read that the carousel data is only the items, that has no `settings` entry? so what about the items, that have `settings` explicitly to set a refresh interval for example?

Reply:
ah ok, i see that each item itself normally do not have a `settings` but the configs directly, like

```
[
  {
    "picture": {
      "uri": "https://cataas.com/cat"
    },
    "message": "<h1 style=\"color: #ffffff; margin-bottom: 0\">CatCäm</h1>",
    "dates": [
      {
        "dateStart": "2021-01-01"
      }
    ],
    "refreshInterval": 5000
  }
]
```

Reply:
`autoplayInterval` is the value that allows the slider to scroll the images after the desired time.

Ok, ich habe den Code aktualisiert und `sliderSettings` zu `globalSettings.settings` hinzugefügt, so dass `homeCarousel` wie bisher verwendet werden kann.